examples/random_module.py

```diff
@@ -1,11 +1,14 @@
 import random
 import os
 import somelib
+import sys
 
 bad = random.random()
 bad = random.randrange()
 bad = random.randint()
 bad = random.choice()
+if sys.version_info.major >= 3 and sys.version_info.minor >= 6:
+    bad = random.choices()
```

Comment:
This code is never actually ran, so there's little reason to have this guard I think?

Reply:
That makes a lot of sense.  Fixed.